main.go

```diff
@@ -77,7 +78,7 @@ func main() {
 
 	_ = godotenv.Load()
 
-	devstackEnvFile := config.DevstackEnvFile()
+	devstackEnvFile := config_legacy.DevstackEnvFile()
```

Comment:
> what is this for?

https://docs.bacalhau.org/references/running-locally#start-the-cluster
> you can set DEVSTACK_ENV_FILE environment variable to the name of a .env file that devstack will write to, and bacalhau commands will read from, before launching the devstack e.g.:

> DEVSTACK_ENV_FILE=.devstack.env bacalhau devstack

> When the devstack is shut down, the local env file (if configured) will be removed. It is suggested you use .devstack.env to avoid clashing with longer lived .env files.

> should we move it from config_legacy?

I left it under legacy as I am not sure if it's something we intend to continue supporting moving forward. Would you prefer it live in the main `config` package instead?